projects/rero/ng-core/src/lib/record/detail/detail.component.ts

```diff
@@ -49,6 +49,15 @@ export class DetailComponent implements OnInit, OnDestroy {
     message: ''
   };
 
+  /**
+   * Record can be used ?
+   */
+  useStatus: ActionStatus = {
```

Comment:
useRecord or useResource?

Reply:
as we already use `deleteStatus` and `udpateStatus`, it makes sense to use `useStatus`. Are you agree ?
I already write a function 'useRecord()' in this component (used when you click on 'use' button.
So a solution, should be to rename all 'status' with 'resource' --> `updateResourceStatus`, `deleteResourceStatus` and  `useResourceStatus`.